HeliPort/Supporting files/itl_80211_state+Description.swift

```diff
@@ -27,7 +27,7 @@ extension itl_80211_state: CustomStringConvertible {
         case ITL80211_S_RUN:
             return "Wi-Fi: Connected"
         default:
-            return "Wi-Fi: Off"
+            return "Wi-Fi: Kext not loaded"
```

Comment:
I think it's better to use "Wi-Fi: Unavailable" because itlwm might have loaded but in an error state, "kext not loaded" seems to be a bit misleading for me.

Reply:
Would you prefer "Wi-Fi: Status unavailable" or "Wi-Fi: Unavailable"? "Wi-Fi: Status unavailable" has localization strings so we don't have to change it, but whichever is fine to me.

Reply:
I would prefer “Status Unavailable” 😄